src/styles/bloks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 .stars__item:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
```

Comment:
Ensure that the path to the `star-active.svg` image is correct and that the image exists at the specified location. This is crucial for the active state to display properly.